src/sentry/identity/oauth2.py

```diff
@@ -102,6 +119,48 @@ def get_oauth_data(self, payload):
 
         return data
 
+    def refresh_identity(self, auth_identity):
```

Comment:
I think I'd change the parameter name from `auth_identity` to just `identity`, since where this code was previously was specific to auth identities (which are still a separate thing)